### DISCUSSION TITLE
Seafood Platter

### DISCUSSION THREAD

**Discussion by Aceplante** [vectorwing/FarmersDelight#695] [Ideas]
Crafted with three cooked calamari, which as of now, is not used in any other recipes and has bad stats on its own. This would potentially fix that.

**Comment by vectorwing** [comment]
FD doesn't have Cooked Calamari; that must be from another mod or add-on.
Also, it already has a seafood platter: Rice Roll Medley, made from salmon/cod/kelp rolls. Try it out, it's delicious! 👌

**Reply by Aceplante** [reply]
Yes, I just realized that was added by cultural delights which is not an official add on.
